### DISCUSSION TITLE
偷資料後聘請厲害律師就穩了？AI 侵權官司的真相

### DISCUSSION THREAD

**Discussion by Tina-Peng-1566** [1101-GS4719-C/Python-Community-GS4719-C#2075] [General]
https://finance.technews.tw/2024/08/31/artists-score-major-win-in-copyright-case-against-ai-art-generators/

先前曾看到有許多文章在討論關於AI侵權的問題，但一直只停留在「煩惱創作者到底該如何保護自己的權益」。沒想到，結論竟是個無奈的現實:科技公司多半會為了訓練自家AI，而先斬後奏--先偷資料，之後再聘請好律師幫忙處理侵權訴訟的相關問題。
因為，因為若該公司的AI廠品尚未成功出品， 創作者不容易發現自己的作品曾被拿去當AI的訓練素材，除非像ChatGPT已經廣為人知，創作者才有可能在因緣際會下發現。
有人說:那就走法律途徑、控告該公司呀!
但實際上沒這麼簡單，尤其是「舉證」。創作者要先從龐大的資料庫中找出自己的作品，並釐清AI究竟是直接複製作品，還是只是將其作為參考資料後，轉化給使用者，最後，還要說明為何事實並非AI科技公司所說的「合理使用」。

若以我的理解，似乎只有消費者和律師是漁翁得利的那一方?
畢竟，律師不管最後是否勝訴，都能拿到相應的報酬，而只要該AI成功推出，消費者就能使用。
但，消費者有一天也可能成為創作者!所以，我認為目前的方法應該是不管是哪一方，都應該設身處地為對方著想，因為誰都不知道，有一天你可能變成其他立場、角色而遇到相同的問題。

若你是AI科技公司的老闆，你的立場與想法為何?
若你是創作者，又會如何看待與思考這個問題呢?
歡迎在下方留言分享你的想法!

**Comment by ds0w0** [comment]
這是一個非常複雜且具有深遠影響的問題，牽涉到道德、法律、技術和經濟等多方面的考量。

如果我是AI科技公司的老闆，理所當然我會面臨來自市場的激烈競爭壓力，尤其是開發出更強大的AI模型和工具。在這種背景下，我可能會傾向於優先考慮如何快速進行模型訓練，以推出市場競爭力更強的產品。然而，我也必須謹慎考慮到這樣的行為是否會侵犯他人的權利，並對公司長期的聲譽和法律風險進行評估。
雖然取得大量資料來訓練模型是技術上的需求，但我也會意識到這涉及到創作者的權利和知識產權問題。我可能會尋找更合乎倫理的方式，例如與創作者合作，提供適當的報酬和授權，而不是單純依賴未經授權的資料來進行訓練。
為了避免法律風險，我可能會考慮提前與律師團隊合作，設計合理的資料使用政策，並在可能的情況下爭取合法使用創作者的作品。這不僅有助於保護公司免受法律訴訟，也能減少與創作者的潛在衝突。

而以創作者的角度思考，作為創作者，我會非常關心自己的作品被未經授權使用的問題，尤其是在AI模型的訓練中。我可能會感到被侵犯，因為我的創意和心血可能在未經同意的情況下被他人用來獲利，這對我的職業生涯和經濟利益都有潛在的威脅。
儘管法律訴訟是保護自己權利的途徑之一，但我也會意識到這個過程的複雜性和艱辛。舉證過程可能需要大量的時間和資源，而且結果也不一定會對我有利。這種現實上的困難讓我思考，是否應該通過其他方式來保護自己的作品，例如使用更嚴格的版權保護工具或與AI公司談判達成授權協議。
我會嘗試理解AI科技發展對社會的廣泛影響，並尋求與科技共存的方式。這可能意味著擴展我的創作形式，將AI視為工具而非威脅，或是參與到AI技術的發展過程中，確保創作者的聲音和權益能夠得到尊重。

基於以上，無論我是AI科技公司老闆還是創作者，我都會鼓勵雙方尋求合作而非對峙的方式。例如，AI公司可以考慮與創作者建立合作關係，提供公平的補償和授權，這樣既能滿足技術需求，也能保護創作者的權益。我會呼籲社會各界共同努力，完善有關AI技術應用的法律與倫理規範，確保在技術進步的同時不損害創作者的合法權益。這是一場持久的戰役，需要平衡技術創新與創作者權益，雙方都應該在彼此理解和尊重的基礎上，尋找出路。
